packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -78,12 +125,49 @@ export class LionProgressIndicator extends LocalizeMixin(LitElement) {
 
   connectedCallback() {
     super.connectedCallback();
-    this.setAttribute('role', 'status');
-    this.setAttribute('aria-live', 'polite');
+    this.setAttribute('role', 'progressbar');
+  }
+
+  /**
+   * Update aria labels on state change.
+   * @param {import('@lion/core').PropertyValues } changedProperties
+   */
+  updated(changedProperties) {
+    super.updated(changedProperties);
+
+    if (!this.indeterminate) {
+      if (changedProperties.has('value')) {
+        if (this.value < this.min) {
+          this.value = this.min;
+          this.setAttribute('aria-valuenow', this.min.toString());
+        } else if (this.value > this.max) {
+          this.value = this.max;
+          this.setAttribute('aria-valuenow', this.max.toString());
+        } else {
+          this.setAttribute('aria-valuenow', this.value.toString());
+        }
+      }
+      if (changedProperties.has('min')) {
+        this.setAttribute('aria-valuemin', this.min.toString());
+        if (this.value < this.min) {
+          this.value = this.min;
+        }
+      }
+      if (changedProperties.has('max')) {
+        this.setAttribute('aria-valuemax', this.max.toString());
+        if (this.value > this.max) {
+          this.value = this.max;
+        }
+      }
+    }
   }
 
   onLocaleUpdated() {
-    const label = localize.msg('lion-progress-indicator:loading');
-    this.setAttribute('aria-label', label);
+    if (
```

Comment:
what happens when the user defines his own aria label (or aria-describedby), after init?